addons/l10n_in/models/template_in.py

```diff
@@ -87,11 +87,11 @@ def _get_in_account_fiscal_position(self):
             },
         }
 
-    def _get_l10n_in_fiscal_tax_vals(self, use_zero_rated_igst=False):
+    def _get_l10n_in_fiscal_tax_vals(self, use_zero_rated_igst=False, use_sez_exp_igst=False):
         return [Command.clear()] + [
             Command.create({
                 'tax_src_id': f"sgst_{tax_type}_{rate}",
-                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}",
+                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst and not use_zero_rated_igst else ''}",
```

Comment:
```suggestion
                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst else ''}",
```

Reply:
Hello @hamo-odoo, I have made the requested changes.